reinforcement_learning/bindings/cs/rl.net.cli/rl.net.cli.csproj

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <PlatformTarget>x64</PlatformTarget>
+    <Platform >x64</Platform>
+    <ProjectGuid>{D0A78C03-F8E9-447F-99D4-795F6E33D8CE}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>RL.Net.Cli</RootNamespace>
+    <AssemblyName>rl</AssemblyName>
+    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <OutputPath>$(SolutionDir)$(Platform)\$(Configuration)</OutputPath>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <!-- What are you?! -->
```

Comment:
why this comment?

Reply:
I do not get what the <Deterministic> property is controlling. I'll get rid of the comment - I wanted to take a second look later, but forgot.